backend/editor/entries.py

```diff
@@ -608,3 +608,14 @@ async def full_text_search(self, text):
         _result = await get_current_transaction().run(query, params)
         result = [record["node"] for record in await _result.data()]
         return result
+
+    async def delete_taxonomy_project(self, branch, taxonomy_name):
+        """
+        Delete taxonomy projects
+        """
+
+        query = f"""
+            MATCH (n:{self.project_name}) DETACH DELETE n
```

Comment:
should'nt you add something like `RETURN count(n)` to have a return value ?

Reply:
Done